### ISSUE TITLE
Calculation in progress/finished indication

### ISSUE THREAD

**Issue by DenoBeno** [clarity-h2020/csis-helpers-module#17]
EMIKAT calculation took 6 minutes for the test study that I tried today. Possibly more for larger areas.

It would be nice to have some indication on the site saying "calculation failed/in progress/done".

**Comment by patrickkaleta** [comment]
### Status update

Most of this feature is already implemented in [this branch](https://github.com/clarity-h2020/csis-helpers-module/tree/feature/017-pull-emikat-status) (still need to update the calculation field of the study after calculation in Emikat has finished + properly phrase the messages for the users). Once it's all done, I will merge and deploy it to our CSIS server.

One problem I encountered is that the `SortBy ` parameter doesn't seem to work on Emikat-side.
![batchjob-order-wrong](https://user-images.githubusercontent.com/42167036/67937266-8d5c1380-fbcd-11e9-8173-415604150b04.png)

Even when you try to access the endpoint via Swagger and set the `SortBy` parameter to **"OBJECT_ID=DESC"**, it will ignore this parameter and return the batchjobs in (for me) no apparent order.

**This is a crucial problem.**  I need the most recent batchjobs first, because I then iterate over them until I find the "initial" batch job that triggered the current (re-)calculation. That's how I separate relevant batch jobs from possibly older batch jobs from previous calculations.

@humerh please take a look at this `sortBy` parameter.

**Comment by patrickkaleta** [comment]
I've finished the implementation and deployed it onto the CSIS server. Users now can see messages like this, when they are in the Study-step and the calculation is ongoing (of course they are not going to see all 3 at once):
![status messages-new](https://user-images.githubusercontent.com/42167036/68082934-22a41580-fe23-11e9-91d7-9a6baa56fedf.png)

More details about this feature in the [readme](https://github.com/clarity-h2020/csis-helpers-module/blob/dev/README.md#pulling-current-calculation-status-from-emikat-for-ongoing-calculations).

**What still needs to be fixed is the problem with the sorting of the batchjobs that I get from Emikat. Only after this sorting is working, the CSIS helpers module will be able to generate meaningful status messages.**

**Comment by patrickkaleta** [comment]
Ok, batch jobs are now always sorted in **ascending** order, however we need them in **descending** order.

@humerh can you please check again if the sorting parameter is handled correctly in Emikat or if the syntax I'm using is correct?

This is an example of a request I'm making: 
https://service.emikat.at/EmiKatTst/api/scenarios/3233/feature/tab.AD_V_BATCH_IN_QUEUE.1710/table/data?rownum=20&filter=SZM_SZENARIO_REF=3233&sortby=OBJECT_ID=DESC

That's the relevant part for the sorting: **...&sortby=OBJECT_ID=DESC**

No matter if I change it to "ASC" or omit it completely, the batch jobs always stay sorted in ascending order.

**Comment by patrickkaleta** [comment]
Thanks to @humerh I now configured the `sortby` parameter correctly and batch jobs are now sorted the way we need them. Closing this issue now.
